frontend/controller/app/group.js

```diff
@@ -155,12 +155,12 @@ export default (sbp('sbp/selectors/register', {
       }
       const promptConfig = enforceDunbar
         ? {
-            heading: 'Large group size',
+            heading: L('Large group size'),
             question: L("Group sizes are limited to {a_}Dunbar's Number{_a} to prevent fraud.", translationArgs),
             primaryButton: L('OK')
           }
         : {
-            heading: 'Large group size',
+            heading: L('Large group size'),
```

Comment:
These are unrelated to the issue itself but I noticed `L()` is missing here, so boyscouted them.